### DISCUSSION TITLE
Opponent rating data hard to understand

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#21] [Q&A] [answered]
It is hard to understand what data this display states for guest players. If there is not data to display, in the table, it should state 'not available' which seems to be what the '-' indicates.

There also is a stray brown box in the lower-left side.

![Screen Shot 2023-05-04 at 10 18 46 AM](https://user-images.githubusercontent.com/4314771/236291227-b880fdf2-b79b-41fc-bfcb-986377ca6c8b.png)
[eagerwords-test.com-1683220761364.log](https://github.com/azadbolour/eagerwords-community/files/11400212/eagerwords-test.com-1683220761364.log)

**Comment by azadbolour** [comment, marked as answer]
I would prefer using an icon of some sort for unavailable data, and the dash is basically a substitute for an icon. Duly noted that the dash as it stands should be improved. Will add that to the backlog. 

The stray brown box will be removed.

Azad
